src/kem/classic_mceliece/kem_classic_mceliece_348864.c

```diff
@@ -6,6 +6,7 @@
 
 #if defined(OQS_ENABLE_KEM_classic_mceliece_348864)
 
+#if defined(OQS_ENABLE_KEM_classic_mceliece_348864)
```

Comment:
Why this change? Duplication?

Reply:
yeah this is superfluous, updated it to avoid the duplication.